packages/pigeon/README.md

```diff
@@ -112,9 +114,9 @@ public interface Api2Host {
 
 ## Null Safety (NNBD)
```

Comment:
We should caveat this a bit more clearly. If we're highlighting null safety more prominently, we should be clear there are 2 layers. There's the general null safety concept and there we're not doing too much yet. Things can't be defined as non-null and the nullable types aren't specifically annotated as so in kotlin and swift. There's also the Dart 2.12 null safety mechanism. We don't support non-null neither but we properly tag everything as nullable now.

Reply:
done